orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -201,7 +201,7 @@ class RunTaskHandler(
                 }
               }
               val status = stage.failureStatus(default = TERMINAL)
-              stage.context["exception"] = exceptionDetails
+              taskModel.taskExceptionDetails["exception"]=exceptionDetails
```

Comment:
What is this change doing? Are parts of Deck not dependent on the exception being at the root of the context rather than under each task?

Reply:
Restored the previous code